tools/building.py

```diff
@@ -920,6 +920,7 @@ def wasm2js(js_file, wasm_file, opt_level, minify_whitespace, use_closure_compil
 
 
 def strip(infile, outfile, debug=False, sections=None):
+  'Strip DWARF and/or other specified sections from a wasm file'
```

Comment:
Use triple double quotes for docstrings.

Reply:
Done